kythe/typescript/indexer.ts

```diff
@@ -88,6 +88,7 @@ export interface IndexerHost {
  * indexer.
  */
 export interface Plugin {
+  /** Name of the plugin. It will be printed in console when running plugin. */
```

Comment:
Can you specify "It will be printed to stderr when running plugin." instead?

Reply:
Done.